src/main/java/frc/trigon/robot/subsystems/collector/CollectorConstants.java

```diff
@@ -0,0 +1,52 @@
+package frc.trigon.robot.subsystems.collector;
+
+import com.ctre.phoenix.motorcontrol.StatorCurrentLimitConfiguration;
+import com.ctre.phoenix.motorcontrol.can.WPI_TalonFX;
+
+public class CollectorConstants {
+    protected static final double
+            COLLECTING_VOLTAGE = 1,
+            OPENING_VOLTAGE = 1,
```

Comment:
these are really low. opening should be 3-4 and collecting 4-5

Reply:
These are placeholder values.

Reply:
Correct but even place holder values can be realistic because we don't need accurate speed for this so just putting something that sounds about right is generally good enough. in addition, when putting place holders, you mustn't put very low values or very high values because if forgotten about high values can be dangerous and low values won't move then you waste time problem solving just to see that it was a stupid mistake